docs: add  docs for cluster adoption (#5218)
spectrocloud/librarium: docs/docs-content/clusters/edge/local-ui/cluster-management/create-cluster.md

You can create and manage single-node as well as multi-node clusters on airgapped hosts locally from Local UI. This page
teaches you how to create a cluster using Local UI.

> Unlike clusters that are managed by Palette, a local cluster is managed locally through Local UI. You can use Local UI
> to create, update, scale and delete the cluster. If your circumstances change and you need the cluster to be managed
> centrally, you also have the option of [pairing your local cluster to a Palette instance](./local-to-central.md) to be
> managed centrally later on.

Reviewer: ```suggestion
Unlike clusters that are managed by Palette, a local cluster is managed locally through Local UI. You can use Local UI to create, update, scale, and delete the cluster. If your situation changes and you need central management of the cluster, you can choose to [pair your local cluster to a Palette instance](./local-to-central.md) for centralized management.
```
Lenny, the last sentence kept tripping me up. I tried to simplify it, take a peek and see what you think :)
